src/modm/platform/gpio/stm32/pin.hpp.in

```diff
@@ -244,7 +244,7 @@ struct Gpio{{ port ~ pin }}::{{ signal.name }}<Peripheral::{{ signal.driver }}>
 		%% if signal.driver.startswith("Adc") and signal.name.startswith("In")
 template<>
 constexpr int8_t
-Gpio{{ port ~ pin }}::AdcChannel<Peripheral::{{ signal.driver }}> = {{ signal.name.replace("In", "") }};
+Gpio{{ port ~ pin }}::AdcChannel<Peripheral::{{ signal.driver }}> = {{ signal.name.replace("In", "")[0] }};
```

Comment:
Note to self: This cuts off double digit channels. (L1 devices have channel `0b` in addition to `0`).